upgrade-ops-manager/vsphere/params.yml

```diff
@@ -36,3 +36,4 @@ ntp: CHANGEME
 opsman_network: CHANGEME
 iaas_type: vsphere
 git_private_key:
+om_disk_type: thick # use thick or thin disk provisioning
```

Comment:
This should be changed to thin given the old default was thin.

The param should also be `opsman_disk_type` given the other params format.